### PR TITLE
Using scratch folder

### PR DESCRIPTION
**Pull request by fonsecareyna82**

### CODE REVIEW

File: pyworkflow/utils/properties.py [hunk 1]
```diff
@@ -387,6 +387,16 @@ class PluginStates:
     ERRORS = 'errors'
 
 
+class PluginInformation:
```
Comment (pconesa) [inline pyworkflow/utils/properties.py:390]: What is all this? This seems to belong to a different task.

Comment (pconesa) [reply pyworkflow/utils/properties.py:390]: I see this is needed by scipion-app?

**Review by pconesa — CHANGES_REQUESTED** [review]
I think we should start asking for the corresponding doc page if the PR requires it. Please, add some page/section mentioning how to use a scratch disc in Scipion

**Comment by pconesa** [comment]
Closes #168

File: pyworkflow/utils/path.py [hunk 2]
```diff
@@ -154,6 +155,25 @@ def makePath(*paths):
...
+    """
+    tmpPath = protocol._getTmpPath()
+    if not os.path.exists(tmpPath) and len(tmpPath):
+        scratchPath = Config.SCIPION_SCRATCH
+
+        if scratchPath is None:  # Case when SCIPION_SCRATCH isn't exist. TMP folder is created
+            os.makedirs(tmpPath)
+        else:
+            folderId = protocol.getProject().getProtWorkingDir(protocol)
+            tmpScratchFolder = os.path.join(scratchPath, folderId)
```
Comment (dmaluenda) [inline pyworkflow/utils/path.py:171]: A couples of projects having the same run name will share the scratch folder. I think it's risky.

Maybe something like `tmpScratchFolder = os.path.join(scratchPath, projectName, folderId)` is more safety.

Comment (pconesa) [reply pyworkflow/utils/path.py:171]: Thanks @dmaluenda , actually we wanted to do that. Somehow we forgot.

Comment (dmaluenda) [reply pyworkflow/utils/path.py:171]: Yeah, I think it is not my genuine idea. I remember I read it somewhere XD

**Review by JorMaister — APPROVED** [review]

File: pyworkflow/utils/properties.py (hunk 1 above)
Comment (fonsecareyna82) [reply pyworkflow/utils/properties.py:390]: It's solved!!

**Review by pconesa — APPROVED** [review]

**Pull request merged**